cookbook/sortAbsoluteInstants.mjs

```diff
@@ -0,0 +1,28 @@
+/**
+ * getSortedInstants will sort an array of strings (each of which is parseable as a Temporal.Absolute and may or may not include an IANA time zone name)
+ * by the corresponding absolute time (e.g., for presenting global log events sequentially).
+ *
+ *
+ * @param {string[]} parseableAbsoluteStrings This is a Date instance
+ * @param {boolean} reverse accending or descending order
+ * @returns {Temporal.Absolute[]} Temporal.Absolute instance
+ */
+function getSortedInstants(parseableAbsoluteStrings, reverse = false) {
+    const absoluteTimes = parseableAbsoluteStrings.map(v => {
+        return Temporal.Absolute.from(v);
+    });
+
+    const sortedAbsoluteTimes = absoluteTimes.sort((a, b) => {
+        return a.compare(b);
+    });
+
+    return reverse ? sortedAbsoluteTimes.reverse() : sortedAbsoluteTimes;
```

Comment:
#240 asks for the _strings_ to be sorted by corresponding absolute time, not for them to be converted into Temporal.Absolute instances first.

Reply:
This is still incorrect; it returns an array of Temporal.Absolute instances rather than an array of the input strings in an order determined by a comparison of Absolute instances derived from them. Like @ptomato mentioned, the expected pattern is decorate-sort-undecorate.